.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -323,7 +323,7 @@ jobs:
         run: |
           kubectl patch node kind-worker3 --type=json -p='[{"op":"add","path":"/metadata/labels/cilium.io~1no-schedule","value":"true"}]'
           kubectl create -n kube-system secret generic cilium-ipsec-keys \
-              --from-literal=keys="3+ rfc4106(gcm(aes)) $(echo $(dd if=/dev/urandom count=20 bs=1 2> /dev/null | xxd -p -c 64)) 128"
+              --from-literal=keys="3+ rfc4106(gcm(aes)) $(dd if=/dev/urandom count=20 bs=1 2> /dev/null | xxd -p -c 64) 128"
```

Comment:
We'll also want to update the documentation to match what runs in CI (even if it shouldn't make the slightest difference). I've sent https://github.com/cilium/cilium/pull/39492 to apply the change everywhere, but if you think it's easier to carry that change in your PR, go ahead :)